src/node/proposals.h

```diff
@@ -54,14 +54,18 @@ namespace ccf
     REJECTED, //< Proposal was rejected by vote, will never be enacted
     FAILED, //< Proposal passed a successful vote, but its proposed actions
             // failed, will never be enacted
+    INVALIDATED, //< Proposal was open when its semantics were potentially
+                 // changed (code or constitution were modified), so it was
+                 // automatically invalidated
```

Comment:
I've just realised this is a potentially confusing name, since one of the constitution functions is `validate` and this is totally unrelated to that concept of validity. Anyone have a better, but still concise, name for "dead because semantics may have changed".

Reply:
Obsolete, out-dated, superannuated, antediluvian?

Reply:
stale?

Reply:
Dropped?